Formula/ciphey.rb

```diff
@@ -207,7 +210,7 @@ def install
     expected_text = "Hello from Homebrew"
     assert_equal shell_output("#{bin}/ciphey -g -t #{input_string}").chomp, expected_text
 
-    python = Formula["python@3.10"].opt_bin/"python3"
+    python = libexec/"bin/python"
     system python, "-c", "from ciphey import decrypt"
     system python, "-c", "from ciphey.iface import Config"
```

Comment:
```suggestion
    system python3, "-c", "from ciphey import decrypt"
    system python3, "-c", "from ciphey.iface import Config"
```
I believe this is meant to test that the `import`s work without having to use the `python` in the `libexec` venv.